locales/th/json.json

```diff
@@ -1,17 +1,17 @@
 {
-    "(and :count more error)": "(and :count more error)",
-    "(and :count more errors)": "(and :count more errors)",
+    "(and :count more error)": "(และข้อผิดพลาดอีกมากกว่า :count รายการ)",
+    "(and :count more errors)": "(และข้อผิดพลาดอีกมากกว่า :count รายการ)",
     "30 Days": "30 วัน",
     "60 Days": "60 วัน",
     "90 Days": "90 วัน",
     ":amount Total": ":Amount ทั้งหมด",
-    ":days day trial": ":Days day trial",
+    ":days day trial": ":Days วันสำหรับทดลองใช้",
     ":resource Details": ":Resource รายละเอียด",
     ":resource Details: :title": ":Resource รายละเอียด::title",
-    "A fresh verification link has been sent to your email address.": "A fresh verification link has been sent to your email address.",
-    "A new verification link has been sent to the email address you provided during registration.": "ใหม่ปลายทางเชื่อมโยงถูกส่งไปที่อยู่อีเมลที่คุณให้ตอบทะเบียนรถด้วย.",
-    "A new verification link has been sent to the email address you provided in your profile settings.": "A new verification link has been sent to the email address you provided in your profile settings.",
-    "A new verification link has been sent to your email address.": "A new verification link has been sent to your email address.",
+    "A fresh verification link has been sent to your email address.": "ลิ้งสำหรับยืนยันตัวตนล่าสุดได้ถูกส่งยังที่อีเมล์ของคุณแล้ว",
```

Comment:
Is missing a full-stop at the end of a sentence grammatically correct in your language?

Reply:
Yes, it's correct. A space is used at the end of the sentence instead of a fullstop.

Reply:
Understood, thanks.